Dockerfile

```diff
@@ -1,6 +1,38 @@
 # syntax=docker/dockerfile:1
 
-FROM technitaur/tribesbot:base as build
+FROM debian:stable-slim as debinstall
+SHELL [ "/bin/bash", "-c" ]
+RUN apt-get update
```

Comment:
Is this run inside of the docker container or on the users machine?

Reply:
It's inside the docker environment, but at build time. So closer to 'inside the container' although that isn't exactly true.

Reply:
All the commands in the Dockerfile are run inside of the container, yes. And by the way, thanks for the compliment!

Reply:
> It's inside the docker environment, but at build time. So closer to 'inside the container' although that isn't exactly true.

Yeah, that's a good point, I forgot about that distinction. So it would probably be most accurate to say that all of these commands are run 'in the image'. And then the resulting image is what we launch the container out of.

Reply:
Gotcha, thank you both for the clarification -- I was just making sure it wasn't being run implicitly on the user's machine.